docs/ingressroute.md

```diff
@@ -198,9 +198,12 @@ IngressRoutes follow a similar pattern to Ingress for configuring TLS credential
 
 You can secure an IngressRoute by specifying a secret that contains a TLS private key and certificate.
 Currently, IngressRoutes only support a single TLS port, 443, and assume TLS termination.
-If the `virtualhost` section includes domain aliases, the certificate must include the necessary Subject Authority Name (SAN) for each alias.
+If multiple IngressRoute's utilize the same secret, then the certificate must include the necessary Subject Authority Name (SAN) for each fqdn.
 Contour (via Envoy) uses the SNI TLS extension to handle this behavior.
 
+Contour also follows a "secure first" approach, by enabling TLS for a vhost, any request to an insecure port (e.g. 80), Contour will configure Envoy to respond to the request with a 301 redirect to the secure interface.
```

Comment:
This sentence is a bit confusing to me. I think the problem is the "Contour will configure Envoy to respond to the request" part.

Maybe something like:

Contour also follows a "secure first" approach. When TLS is enabled for a virtual host, any request to the insecure port is redirected to the secure interface with a 301 redirect.